libraries/classes/Controllers/Server/BinlogController.php

```diff
@@ -44,33 +44,27 @@ public function __construct(ResponseRenderer $response, Template $template, Data
 
     public function __invoke(ServerRequest $request): void
     {
-        $GLOBALS['errorUrl'] = $GLOBALS['errorUrl'] ?? null;
+        $log = $request->getParsedBodyParam('log');
+        $position = (int) $request->getParsedBodyParam('pos', 0);
 
-        $params = [
-            'log' => $_POST['log'] ?? null,
-            'pos' => $_POST['pos'] ?? null,
-            'is_full_query' => $_POST['is_full_query'] ?? null,
-        ];
         $GLOBALS['errorUrl'] = Url::getFromRoute('/');
 
         if ($this->dbi->isSuperUser()) {
             $this->dbi->selectDb('mysql');
         }
 
-        $position = ! empty($params['pos']) ? (int) $params['pos'] : 0;
-
         $urlParams = [];
-        if (isset($params['log']) && array_key_exists($params['log'], $this->binaryLogs)) {
-            $urlParams['log'] = $params['log'];
+        if (array_key_exists($log, $this->binaryLogs)) {
+            $urlParams['log'] = $log;
         }
 
         $isFullQuery = false;
-        if (! empty($params['is_full_query'])) {
+        if (! empty($request->getParsedBodyParam('is_full_query'))) {
```

Comment:
```suggestion
        if ($request->getParsedBodyParam('is_full_query')) {
```

The `is_full_query` parameter is a flag and always has the value '`1`' when present. So, `if ($request->hasBodyParam('is_full_query')) {` will also work here.

You can see it been set here:
https://github.com/phpmyadmin/phpmyadmin/blob/97aac39a40cef8f0ec63f380be79e14c4633bd67/libraries/classes/Controllers/Server/BinlogController.php#L67-L71

And been unset here:
https://github.com/phpmyadmin/phpmyadmin/blob/97aac39a40cef8f0ec63f380be79e14c4633bd67/libraries/classes/Controllers/Server/BinlogController.php#L88-L91